mobile/lib/ui/viewer/file/file_bottom_bar.dart

```diff
@@ -61,7 +62,7 @@ class FileBottomBarState extends State<FileBottomBar> {
   @override
   Widget build(BuildContext context) {
     return _isFileSwipeLocked
-        ? IgnorePointer(child: Container())
+        ? const IgnorePointer(child: SizedBox.shrink())
```

Comment:
Is this `IgnorePointer` needed?

Reply:
It prevent its subtree from receiving pointer events.